WcaOnRails/app/views/registrations/edit_registrations.html.erb

```diff
@@ -9,16 +9,17 @@
       <% else %>
         <h2>Approved registrations</h2>
       <% end %>
-      <% registrations = @competition.registrations.send(status).order(created_at: :asc, id: :asc) %>
-      <%= wca_table table_class: "registrations-table selectable-rows" do %>
+      <% registrations = @competition.registrations.send(status) %>
+      <%= wca_table table_class: "registrations-table", data: { toggle: "table", sort_name: "registration-date-#{status}"  , select_item_name: "selected_registrations[]", click_to_select: "true" } do %>
```

Comment:
nit: There's some extra space before the comma after "#{status}"

Neat find with that click_to_select!